src/Homework3/Publisher.java

```diff
@@ -0,0 +1,34 @@
+package Homework3;
+
+public class Publisher {
+    int ID_pub;
+    String PubName;
+    String PubAddress;
+
+    Publisher() {
```

Comment:
Another empty constructor :) Why?

Reply:
It was added "just in case", deleted it.